components/contribution-flow/StepPayment.js

```diff
@@ -108,6 +120,12 @@ const getPaymentMethodMetadata = pm => {
   }
 };
 
+/**
+ * Returns true is a payment method is expired
+ * @param {string} expiryDate
+ */
+const paymentMethodExpired = expiryDate => expiryDate && moment(new Date(expiryDate)).isBefore(moment());
```

Comment:
Before `moment` is a heavy dependency we prefer to avoid it when we can,

Here it's easy to do this instead

```suggestion
const paymentMethodExpired = expiryDate => expiryDate && new Date(expiryDate) < new Date();
```

Reply:
Okay